setup.py

```diff
@@ -47,6 +47,7 @@
         'systemd-python',
         'Jinja2>=2.10',
         'PyGObject',
+        'psutil'
```

Comment:
psutil dependency introduced by this PR:https://github.com/sonic-net/sonic-host-services/pull/103

+Qi as reviewer for viz because #103 approved by Qi.